### PR TITLE
docs(feat): Armory Agent for Kubernetes

### PR DESCRIPTION
**Pull request by aimeeu**
The aim of this PR is to provide just enough information for experienced K8s and Spinnaker users to get started using the Armory Agent service and plugin.  Subsequent PRs can provide additional design and installation content.

Reviewers: please verify that the Quick Start guide has enough info for experienced users

### CODE REVIEW

File: content/en/docs/armory-agent/_index.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent for K8s"
+description: >
+  A native Kubernetes service that enables high-scale deployments to Kubernetes using Spinnaker<sup>TM</sup>
+weight: 20
+---
+
+Armory Agent is a new, flexible way for Spinnaker to interact with your Kubernetes infrastructure that solves the scaling issues plaguing Spinnaker's Kubernetes V2 Provider.
```
Comment (ncknt) [inline content/en/docs/armory-agent/_index.md]: I'd remove any negative reference to the V2 provider. (e.g. `issues plaguing...`). We're not building in opposition to the V2 provider, we're extending to offer more usage (account management flexibility, greater scale). The v2 provider is getting better on its own and it cannot be the agent's only value prop.

File: content/en/docs/armory-agent/_index.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+---
+title: "Armory Agent for Kubernetes"
+linkTitle: "Armory Agent for K8s"
+description: >
+  A native Kubernetes service that enables high-scale deployments to Kubernetes using Spinnaker<sup>TM</sup>
```
Comment (ncknt) [inline content/en/docs/armory-agent/_index.md]: Do we really the ™? We don't put it anywhere else and it looks ... odd.

Comment (317brian) [reply content/en/docs/armory-agent/_index.md]: Putting ™ on at least the first mention of Spinnaker is one of the requirements for using the Spinnaker trademark based on the discussion between the Linux Foundation lawyers and ours. We need to have it on every page for the first mention of Spinnaker. We haven't had the time to work through all the docs yet, but we're starting to put it on every page going forward.

Comment (aimeeu) [reply content/en/docs/armory-agent/_index.md]: I removed the TM thing from the desc in the front matter and stuck it someplace in the page content.

**Comment by aimeeu** [comment]
Replaced by #240

**Pull request closed without merging**